incubator/elasticsearch/templates/client-svc.yaml

```diff
@@ -18,6 +18,10 @@ spec:
     - name: http
       port: 9200
       targetPort: http
+    - name: {{ .Values.prometheusExporter.clientName }}
```

Comment:
Same here: the service port can be conditionally exposed with `Values.prometheusExporter.enabled`